teku/src/main/java/tech/pegasys/teku/cli/options/DepositOptions.java

```diff
@@ -39,9 +50,15 @@ public int getEth1LogsMaxBlockRange() {
   }
 
   public void configure(final TekuConfiguration.Builder builder) {
+    ArrayList<String> mergedEth1Endpoints = new ArrayList<>();
+    if (eth1Endpoint != null) mergedEth1Endpoints.add(eth1Endpoint);
```

Comment:
Our coding style is to always use braces on if statements.  Or we could just use `Optional` here:
```suggestion
    Optional.ofNullable(eth1Endpoint).ifPresent(mergedEth1Endpoints::add);
```

Reply:
sooner or later I'll get your braces code style